### PR TITLE
install CNI plugin via an init container

### PR DESCRIPTION
**Pull request by phoracek**
There is no need to keep the container installing the CNI running forever. We
can use an init container to drop it to the host filesystem and then exit.

Depends on: #112 #111 

/hold
/release-note-none

Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

File: manifests/ovs-cni.yml.in [hunk 1]
```diff
@@ -24,25 +24,28 @@ spec:
       - key: node-role.kubernetes.io/master
         operator: Exists
         effect: NoSchedule
-      containers:
+      initContainers:
       - name: ovs-cni-plugin
         image: ${OVS_CNI_PLUGIN_IMAGE_REPO}/${OVS_CNI_PLUGIN_IMAGE_NAME}:${OVS_CNI_PLUGIN_IMAGE_VERSION}
+        command: ['cp', '/ovs', '/host/opt/cni/bin/macvtap']
```
Comment (phoracek) [inline manifests/ovs-cni.yml.in:30]: Bad copy-paste!

File: manifests/ovs-cni.yml.in [hunk 2]
```diff
@@ -24,9 +24,10 @@ spec:
       - key: node-role.kubernetes.io/master
         operator: Exists
         effect: NoSchedule
-      containers:
+      initContainers:
       - name: ovs-cni-plugin
         image: ${OVS_CNI_PLUGIN_IMAGE_REPO}/${OVS_CNI_PLUGIN_IMAGE_NAME}:${OVS_CNI_PLUGIN_IMAGE_VERSION}
+        command: ['cp', '/ovs', '/host/opt/cni/bin/ovs']
```
Comment (qinqon) [inline manifests/ovs-cni.yml.in:30]: Can se use ${CNI_MOUNT_PATH} here ?
```
'cp', '/ovs', '/host/${CNI_MOUNT_PATH}
```

File: manifests/ovs-cni.yml.in [hunk 3]
```diff
@@ -40,6 +41,7 @@ spec:
         volumeMounts:
         - name: cnibin
           mountPath: /host/opt/cni/bin
```
Comment (qinqon) [inline manifests/ovs-cni.yml.in:43]:
```
/host/${CNI_MOUNT_PATH}
```

File: examples/ovs-cni.yml [hunk 4]
```diff
@@ -24,9 +24,10 @@ spec:
...
         operator: Exists
         effect: NoSchedule
-      containers:
+      initContainers:
       - name: ovs-cni-plugin
         image: quay.io/kubevirt/ovs-cni-plugin:latest
+        command: ['cp', '/ovs', '/host/opt/cni/bin/ovs']
         imagePullPolicy: IfNotPresent
         resources:
           requests:
```
Comment (qinqon) [inline examples/ovs-cni.yml:33]: We don't need this for the init container I think

**Pull request merged**